src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <section class="productCard">
+      <section class="productCard__card card" data-qa="card">
+        <div class="card__image image">
+        </div>
+        <h3 class="card__title title">
+          APPLE A1419 iMac 27" Retina 5k Monoblock (MNED2UA/A)
+        </h3>
+        <div class="card__code code">Product code: 195434</div>
+        <div class="card__review review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="smallFont">Reviews: 5</div>
+        </div>
+        <div class="card__cost cost">
+          <div class="mediumFont">Price:</div>
+          <div class="largeFont">$2,199</div>
+        </div>
+        <div class="card__button button">
+          <a
+            href="#"
+            class="link"
+            data-qa="hover"
+          >
+            <div class="label">BUY</div>
```

Comment:
If you are using this as an BEM Block then it's ok, other wise you may consider give it class as an BEM Element

Reply:
it was damn confusing part and It took me long time to decide how to implement the link and button correctly,
now I used a simple <a> and style the whole button and it's background as well as :hover effect by it. 
it's very nice and pleasant